[D[C[D

```diff
@@ -0,0 +1,49 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
you can remove this file and the next

Reply:
You should generate a new key and not use this one any more

Reply:
I have made a new one. I don't know how this happened. Thank you for informing me.